guides/03_building-with-blocks/02_controlling-layout.md

```diff
@@ -57,8 +57,10 @@ Both Components and Layout elements have a `visible` argument that can set initi
 $code_blocks_form
 $demo_blocks_form
 
+## Variable Number of Outputs
```

Comment:
I received 3 separate questions about how to do this today, suggesting that this might not be very visible in the docs or searchable by Google. Therefore separating this out to be its own section which follows logically from the previous section

Reply:
I wonder if variable outputs should also be a type of interface in the 4 (i guess 5 now) types of interfaces guide. Something to think about